src/components/molecules/GalleryCard/GalleryCard.tsx

```diff
@@ -0,0 +1,23 @@
+import Image from "next/image";
+
+type GalleryImageProps = {
+  src: string;
+  description?: string;
+};
+
+const GalleryImage = ({ src, description }: GalleryImageProps) => {
+  return (
+    // <div>
+    <Image
+      src={src}
+      alt="nature"
+      width={0}
+      height={0}
+      sizes="100vw"
+      style={{ width: "100%", height: "100%" }}
```

Comment:
Let's not use inline styling please. Why did you set `width` & `height` to 0 above?

Reply:
Cause i don't have de widths and heights and next js requires them for their image component. So i thought i'd be better to do it that way for now, until i find a better solution. Thanks for reminding me.

Reply:
Then maybe stick to a plain `<img>` tag?

Reply:
Your call, it's a fair point after all. My point was that next js recommends those for better loading times. I can get the the inline style out though and make it a simple image i guess.